### DISCUSSION TITLE
Does prisma support reverse engineering ?

### DISCUSSION THREAD

**Discussion by surajkrishan** [prisma/prisma#17479] [Q&A] [answered]
Hi team,

I have a question by default prisma generate types from prisma.schema

Is it possible to do the opposite i.e from typescript types -> generate a prisma.schema file ?

**Comment by nurul3101** [comment, marked as answer]
Hello @surajkrishan 👋 

I don't think that is possible at the moment.
There are various [community generators](https://www.prisma.io/docs/concepts/components/prisma-schema/generators#community-generators), but there isn't one that converts types to schema.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
